ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -187,27 +187,25 @@ class JITModBuilder {
     this->FAddrKeys = FAddrKeys;
     this->FAddrVals = FAddrVals;
     this->FAddrLen = FAddrLen;
+
+    JITMod = new Module("", AOTMod->getContext());
```

Comment:
Is it safe to have an empty module name?

Reply:
Yep.